documentation/slate/source/includes/realtime.md

```diff
@@ -22,6 +22,23 @@ For each one of these effects, here's how the Navitia responses will be affected
 A disruption is present in the response of the endpoints described if the request is made during its application period.
 </aside>
 
+
+## <a name="PT_object_collections_data_freshness"></a>Public transport object collections
+
+Several public transport objects have separate collections for `base_schedule` and `realtime`.
+So the data_freshness parameter may affect the number of objects returned depanding on the request.
+
+For exemple when looking for a specific circulation with the collection vehicle_journey with the request like
+`http://api.navitia.io/v1/coverage/<toto>/vehicle_journeys?since=20191008T100000&until=20191008T200000&data_freshness=base_schedule`.
+A vehicle_journey circulating between since and until that is fully deleted by a disruption will
```

Comment:
why end of line here?

Reply:
To stay under 113 char while keeping it visible that the sentence does keep on :smiley: 
If you think I should change I can.